src/libraries/System.Memory.Data/src/System.Memory.Data.csproj

```diff
@@ -4,8 +4,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nullable>enable</Nullable>
     <IsPackable>true</IsPackable>
```

Comment:
Huh, why did this package never showed errors when publishing the nupkgs? It's been turned on all this time but we never saw versioning conflicts?

Reply:
IsPackable doesn't mean it builds the package.  `GeneratePackageOnBuild` is what determines that in Runtime servicing.

Reply:
Gah yeah, sorry. Good point.